pkg/utils/docker/image.go

```diff
@@ -56,6 +58,16 @@ func GetImageTagFromEnv(envName string) (tag string) {
 	return
 }
 
+// get docker pull secrets from environment variables, if it's not existed, return ""
+func GetImagePullSecretsFromEnv(envName string) (pullSecretsStr string) {
+	if value, existed := os.LookupEnv(envName); existed {
+		if existed && len(value) > 0 {
```

Comment:
I think it's a repeated `existed=true` check here

Reply:
> I think it's a repeated `existed=true` check here

thank you, done